# Array of Detection2D messages

Header 				header					# Standard message header.
Detection2D[] 		detections				# A list of the detected objects.
